test/blackbox-tests/test-cases/dynamic-dependencies/read-macro-produces-dyn-deps.t

```diff
@@ -0,0 +1,41 @@
+Tests for dynamic dependencies computed from the `%{read:...}` family of macros
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.0)
+  > EOF
+
+Define 2 rules and a file containing their paths
+
+  $ cat > deps.d <<EOF
```

Comment:
It seems better to produce `deps.d` by a rule to show that the dependency is truly dynamic.